src/styles/card.css

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
```

Comment:
Just add `display: block` to your `img` and `a` and center them by using width and `margin: 0 auto` instead of making your card flexed

Reply:
@OlyaNikora 
Hey! You haven't resolved this comment.